mpu6050_scripts/get_temperature.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if i2cset -y 0 0x68 0x6b 0 ; then
+	sleep 0.2
+else
+	echo "Error occurred!"
+	exit
+fi
+
+while :
+do
+	HIGH=$(i2cget -y 0 0x68 0x41)
+	LOW=$(i2cget -y 0 0x68 0x42)
+
+	SIGN=$(( (HIGH & 0x80) >> 7 ))
+	TEMP=$(( (HIGH << 8) + LOW ))
+
+	if [ $SIGN -eq 1 ] ; then
+		TEMP=$(( -(TEMP ^ 0xFFFF) ))
```

Comment:
This conversation from negative 2's complement to 0x10000 is not correct.
Correct value should be -(0x10000 - TEMP)
It is the same like TEMP - 0x10000.

Reply:
Issue is fixed